browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2176,7 +2176,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			'zoteroaddnote': 'addcitationnote',
 			'zoterorefresh': 'updateall',
 			'zoterounlink': 'unlinkcitation',
-			'zoteroaddeditbibliography': 'addeditbibliography',
```

Comment:
I see that `zoteroaddeditbibliography` is also used somewhere, are you sure we can remove it? shouldn't we keep 2 of them?

Reply:
@eszkadev i have not seen zoteroaddeditbibliography at any place i did test with NC intergration

Reply:
`git grep zoteroaddeditbibliography` shows usage, maybe it is only shown under some conditions
do you have zotero configured? (API token)

Reply:
No i do not have zotero API token. let me try with NC integration with local branding

Reply:
@eszkadev just tested with local theme with Zotero it is working okay. We are good to replace this keyword

Reply:
no, please leave it. If it is here and somewhere is used in the code - let's trust we need it.

grep:
```
browser/src/control/Control.JSDialogBuilder.js:                 'zoteroaddeditbibliography': 'addeditbibliography',
browser/src/control/Control.Menubar.js:                         {name: _('Add Bibliography'), id: 'zoteroaddeditbibliography', type: 'action', hidden: !window.zoteroEnabled},
browser/src/control/Control.NotebookbarWriter.js:                                       'id': 'zoteroaddeditbibliography',
browser/src/control/Control.NotebookbarWriter.js:                                       'class': 'unozoteroaddeditbibliography',
browser/src/docdispatcher.ts:           this.actionsMap['zoteroaddeditbibliography'] = function () {
```

Reply:
Sure i have added the original value and also the corrected one.